test/java/build.gradle

```diff
@@ -0,0 +1,95 @@
+/*
+ * This build file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java Library project to get you started.
+ * For more details take a look at the Java Libraries chapter in the Gradle
+ * user guide available at https://docs.gradle.org/4.0.1/userguide/java_library_plugin.html
+ */
+
+// Apply the java-library plugin to add support for Java Library
+apply plugin: 'java-library'
+apply plugin:'application'
+
+mainClassName = "Test"
+
+// // In this section you declare where to find the dependencies of your project
+// repositories {
+//     // Use jcenter for resolving your dependencies.
+//     // You can declare any Maven/Ivy/file repository here.
+//     jcenter()
+// }
+
+repositories {
+    mavenLocal()
+    maven {
+        url "https://dcartifactory.service.dev/artifactory/sbos-csp-libs/"
+    }
+    maven {
+        url "https://dcartifactory.service.dev/artifactory/csp-platform"
+    }
+    maven {
+        url "https://dcartifactory.service.dev/artifactory/plugins-release"
+    }
+    maven {
+        url "https://dcartifactory.service.dev/artifactory/jcenter"
+    }
+    maven {
+        url "http://repo1.maven.org/maven2"
+    }
+    jcenter {
+        url "http://jcenter.bintray.com/"
+    }
+}
+
+task generateSysl(type:Exec) { 
+   //on linux'
+  commandLine 'python', '-m', 'sysl.reljam',  '--out', 'src/main/java', 'model', '/tuplecomplex', 'UserFormComplex'
```

Comment:
I added this because gradle could not find 'reljam' after doing 'pip install' in my local travis docker image. Not sure why. 
Anyway, things are working now.